shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -308,4 +308,35 @@ - (void)logCapabilityConfigurationWarningIfNeeded:(SEL)selector {
   }
 }
 
+#pragma mark - State Restoration
+
+- (BOOL)application:(UIApplication*)application shouldSaveApplicationState:(NSCoder*)coder {
+  [coder encodeInt64:self.lastAppModificationTime forKey:@"mod-date"];
```

Comment:
Optional: not used much but consider yanking this to a const. Sadly, can't use constexpr since it's an NSString.

Reply:
Well it is duplicated across the file so I think it's a good idea.

Reply:
Done.